cache/src/main/java/com/playone/mobile/cache/db/mapper/PlayoneMapper.kt

```diff
@@ -0,0 +1,34 @@
+package com.playone.mobile.cache.db.mapper
+
+import android.content.ContentValues
+import android.database.Cursor
+import com.playone.mobile.cache.db.Db
+import com.playone.mobile.cache.model.CachedBufferoo
+
+/**
+ * Maps a [CachedBufferoo] instance to a database entity.
```

Comment:
`[CachedBufferoo]` should be `CachedPlayone`.

Reply:
Oh.. as I mentioned, just ignore cache layer for now..